mgmtfn/k8splugin/contivk8s/clients/network.go

```diff
@@ -86,7 +86,7 @@ func (c *NWClient) AddPod(podInfo interface{}) (*cniapi.RspAddPod, error) {
 		if err != nil {
 			return nil, err
 		}
-		return &data, fmt.Errorf("Internal Server Error")
+		return &data, fmt.Errorf("internal Server Error")
```

Comment:
lower case all words?

Reply:
See the other comments in this PR discussion.  This makes error messages composable